src/gtc/dace/expansion/daceir_builder.py

```diff
@@ -150,7 +153,7 @@ def all_regions_same(scope_nodes):
 class DaCeIRBuilder(NodeTranslator):
     @dataclass
     class GlobalContext:
-        library_node: StencilComputation
+        library_node: "StencilComputation"
```

Comment:
If you import `from __future__ import annotations` do you need the quotes? https://peps.python.org/pep-0563/